src/azure-cli/azure/cli/command_modules/appservice/commands.py

```diff
@@ -144,6 +144,7 @@ def load_command_table(self, _):
         g.custom_command('unbind', 'unbind_ssl_cert', validator=validate_app_or_slot_exists_in_rg)
         g.custom_command('delete', 'delete_ssl_cert', exception_handler=ex_handler_factory())
         g.custom_command('import', 'import_ssl_cert', exception_handler=ex_handler_factory(), is_preview=True)
+        g.custom_command('create-managed-certificate', 'create_managed_ssl_cert', exception_handler=ex_handler_factory(), is_preview=True)
```

Comment:
@yonzhan / @Juliehzl can you comment if this naming follows the recommended naming convention
@madsd  i was thinking if we should call this create simply & have another property that defaults to Managed certs as default for now so that we can re-purpose the same command to create AppService certificates when we are ready to add that support as well? What do you think?  If you agree , we can run this by Yutang as well as the PM for this feature.

Reply:
@yutanglin16 - FYI